curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/645f764c37de33015ded1273.md

```diff
@@ -0,0 +1,305 @@
+---
+id: 645f764c37de33015ded1273
+title: Step 73
+challengeType: 0
+dashedName: step-73
+---
+
+# --description--
+
+You can create a constant called `avatar` and set it to the `user` object's `avatar_template`. This `avatar_template` is a string that has a special placeholder `{size}` in it. To customize the avatar's size, you can replace this `{size}` with the value `30`. This way, you can control the size of the user's avatar in your application.
+
+# --hints--
+
+You should create a constant named `avatar` and assign it the value of the `user` object's property `avatar_template` with the placeholder `{size}` replaced with `30`.
+
+```js
+assert(code.match(/const\s+avatar\s*=\s*user\.avatar_template\.replace\(\s*\/{size}\/\s*,\s*30\s*\)/));
```

Comment:
```suggestion
assert(code.match(/const\s+avatar\s*=\s*user\.avatar_template\.replace\(\s*([/'"]){size}\1\s*,\s*30\s*\)/));
```
Test is expecting to use regex, but string will work in here in the same way.

Reply:
added